packages/react-components/src/components/RaiseHandButton.tsx

```diff
@@ -67,7 +67,7 @@ export const RaiseHandButton = (props: RaiseHandButtonProps): JSX.Element => {
   return (
     <ControlBarButton
       {...props}
-      className={mergeStyles(styles, props.styles)}
+      styles={concatStyleSets(styles, props.styles)}
```

Comment:
This change is to prevent a 'Maximum call stack size exceeded' error when returning from a breakout room to main meeting